src/skydb.ts

```diff
@@ -45,7 +45,7 @@ export async function setJSON(
     // fetch the current value to find out the revision.
     let entry: SignedRegistryEntry;
     try {
-      entry = await promiseTimeout(this.registry.getEntry(publicKey, dataKey), 5000);
+      entry = await promiseTimeout(this.registry.getEntry(publicKey, dataKey), 30_000);
```

Comment:
it will not stop the request so we might end up with hundreds of pending requests from one app, the timeout should not be in the client

if this is not possible right now, we should use axios `timeout` option, I **think** it will abort the request on timeout but I am not sure